python/CHANGELOG.rst

```diff
@@ -4,12 +4,19 @@
 
 **Features**
 
+- Added ``Tree.generate_star`` static method to create star-topologies (:user:`hyanwong`,
+  :pr:`934`).
+
 - Added ``equals`` method to TableCollection and each of the tables which
   provides more flexible equality comparisons, for example, allowing
   users to ignore metadata or provenance in the comparison.
   (:user:`mufernando`, :user:`jeromekelleher`, :issue:`896`, :pr:`897`,
   :issue:`913`, :pr:`917`).
 
+- Added ``TableCollection.equals``, a method that compares two table 
```

Comment:
This is already in the changelog

Reply:
Good spot, sorry. I obviously didn't rebase properly. Done now.